src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/clients/CcdCaseEventConsumerTest.java

```diff
@@ -82,11 +82,29 @@ void given_session_is_accepted_when_receiver_complete_throws_error_on_both_calls
 
         doThrow(new ServiceBusException(new Exception(), ServiceBusErrorSource.UNKNOWN))
                 .when(receiverClient).complete(receivedMessage);
+        doThrow(new ServiceBusException(new Exception(), ServiceBusErrorSource.UNKNOWN))
+            .when(receiverClient).abandon(receivedMessage);
+
 
         underTest.consumeMessage(sessionReceiverClient);
 
-        verify(receiverClient, Mockito.times(2)).complete(receivedMessage);
-        verify(receiverClient, Mockito.times(0)).abandon(any());
+        verify(receiverClient, Mockito.times(1)).complete(receivedMessage);
+        verify(receiverClient, Mockito.times(1)).abandon(receivedMessage);
+        verify(receiverClient, Mockito.times(0)).deadLetter(any(), any());
+    }
+
+    @Test
+    void given_session_is_accepted_when_handling_message_throws_error() {
```

Comment:
can we rename these test to follow the should_  pattern? the given syntax does not read completely clearly as there is not then to qualify the outcome?

Reply:
All the ASB service test classes follow the patter of given_session_is